src/main/java/frc/team2412/robot/util/AmpAlign.java

```diff
@@ -0,0 +1,100 @@
+package frc.team2412.robot.util;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.path.GoalEndState;
+import com.pathplanner.lib.path.PathConstraints;
+import com.pathplanner.lib.path.PathPlannerPath;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team2412.robot.subsystems.DrivebaseSubsystem;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+import java.util.List;
+
+public class AmpAlign {
```

Comment:
(For later): Since this reuses a lot of logic from TrapAlign, it would be better to refactor this so the logic is only used in one place.

Reply:
Ok will fix that later